templates/payment/form-fpx.php

```diff
@@ -0,0 +1,30 @@
+<?php if ( ! empty( $viewData['fpx_banklist'] ) ) : ?>
+	<fieldset id="omise-form-installment">
+		<ul class="omise-banks-list">
+			<?php foreach ( $viewData['fpx_banklist'] as $bank ) : ?>
+				<li class="fpx item">
+				<?php if ( $bank['active'] ): ?>
+					<input id="<?php echo $bank["code"]; ?>" type="radio" name="source[bank]" value="<?php echo $bank["code"]; ?>"/>
+					<?php endif; ?>
+					<label for="<?php echo $bank["code"]; ?>" <?php if(! $bank['active']) { echo "class='offline'"; } ?> >
```

Comment:
this part is a little confusing for me. Can this be elseif? Also please fix the indentations on this file